lms/djangoapps/teams/static/teams/js/spec/views/teams_tab_spec.js

```diff
@@ -39,7 +57,8 @@ define([
 
         describe('Navigation', function () {
             it('does not render breadcrumbs for the top level tabs', function() {
-                var teamsTabView = createTeamsTabView();
+                var requests = AjaxHelpers.requests(this),
```

Comment:
I've moved the logic into ``createTeamsTabView`` which is nice because it makes the tests more concise. I'm not a big fan of global variables for understandability, but having a global ``requests`` variable seems reasonable.